packages/browser/src/tracing/browserTracingIntegration.ts

```diff
@@ -213,7 +214,11 @@ export const browserTracingIntegration = ((_options: Partial<BrowserTracingOptio
     startTrackingINP();
   }
 
-  if (enableLongAnimationFrame && PerformanceObserver.supportedEntryTypes.includes('long-animation-frame')) {
+  if (
+    enableLongAnimationFrame &&
+    GLOBAL_OBJ.PerformanceObserver &&
```

Comment:
note for future: cannot reference `PerformanceObserver` since the reference is not defined: https://github.com/getsentry/sentry-javascript/actions/runs/10286131917/job/28466264438